packages/fury-adapter-oas3-parser/lib/parser/parseYAML.js

```diff
@@ -106,8 +106,17 @@ function parse(source, context) {
   try {
     ast = yaml.compose(source);
   } catch (error) {
+    let message;
+
+    if (error.problem) {
+      const problem = error.problem.replace('\t', '\\t');
```

Comment:
Since a tab isn't clear in the error message (it would just show as whitespace), I am replacing tabs specifically the literal value '\t'. The error message would read as:

> error: YAML Syntax: found character \t that cannot start any token, while scanning for the next token - line 1452

Reply:
Should we be aware of other symbols in the error messages, or is it just tabs?

Reply:
Perhaps, tab is the only one I'm aware of so far. Other types of errors from the YAML parser which would cause similiar problems haven't come up yet.

Maybe there is a generic way we can make a representation of a string to expose all unreadable characters.